etc-hosts.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+- name: Update /etc/hosts in guests
```

Comment:
I think this is really clever.. The only thing I can think of that would be a _possible_ alternative would be to do a custom fact script that would manage the logic. That said.. you'd still need to use those facts to populate this file, so I suspect this is probably still the optimal solution.